browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -149,6 +149,31 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_BRAVE_SHIELDS_HTTPS_EVERYWHERE_SWITCH" desc="Switch for HTTPS Everywhere.">
         Upgrade connections to HTTPS
       </message>
+
```

Comment:
why do we need that bunch of empty strings?

Reply:
removed